scripts/delete-old-app-engine-versions.sh

```diff
@@ -52,6 +52,13 @@ check_jq_installed() {
     fi
 }
 
+# ensure that gcloud is installed
+check_gcloud_installed() {
+    if ! gcloud --version 1>/dev/null 2>&1; then
+        abort "gcloud is required; install using brew install google-cloud-sdk"
```

Comment:
Maybe link to https://cloud.google.com/sdk/docs/install instead? Some devs may not be using Homebrew or even macOS.